tools/depends/target/libcec/Makefile

```diff
@@ -20,19 +21,19 @@ $(TARBALLS_LOCATION)/$(ARCHIVE):
 $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	rm -rf $(PLATFORM); mkdir -p $(PLATFORM)
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
-	cd $(PLATFORM); $(AUTORECONF) -vif
-	cd $(PLATFORM); $(CONFIGURE)
+	cd $(PLATFORM); rm -rf build; mkdir -p build
```

Comment:
That doesn't do anything (at least the rm -rf) because the first line of the target already revmoves the whole platform dir - might aswell add the mkdir of build in that line directly ...